### DISCUSSION TITLE
Is there technical report or paper of pyannote.audio v2.0 ??

### DISCUSSION THREAD

**Discussion by Jasonlee1995** [pyannote/pyannote-audio#1112] [General]
Hi!

I have a question about is there any technical report or paper about `pyannote.audio v2.0`?
(if not, have future plan?)

I found the paper of [`pyannote.audio v1.1`](https://arxiv.org/abs/1911.01255) on arXiv.

<img width="700" alt="image" src="https://user-images.githubusercontent.com/49643709/197650058-b960e991-ba72-4d58-b347-2a9e80563802.png">

But as mentioned in README.md, there may be huge difference between `v1.1` and `v2.0` so reading the paper of `v1.1` would be not proper to understand this current repo.

Besides, thanks for the nice work.

**Comment by hbredin** [comment]
Not yet, but something is cooking. Will update this discussion when time comes.

**Reply by hbredin** [reply]
A technical report is available [here](https://huggingface.co/pyannote/speaker-diarization#technical-report).
